geomesa-nifi-processors/src/main/scala/org/geomesa/nifi/geo/validators/ConverterValidator.scala

```diff
@@ -0,0 +1,21 @@
+package org.geomesa.nifi.geo.validators
+
+import org.apache.nifi.components.{ValidationContext, ValidationResult, Validator}
+import org.locationtech.geomesa.convert.{ConfArgs, ConverterConfigResolver}
+
+/**
+  * Simple validator of a convert spec. Tries to parse it to make sure that it can.
+  */
+class ConverterValidator extends Validator{

```

Comment:
you could make this implementation an object - it should be thread safe to re-use